core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -293,6 +296,56 @@ object EnumEventDescriptorV0TLV extends TLVFactory[EnumEventDescriptorV0TLV] {
   }
 }
 
+trait NumericEventDescriptor extends EventDescriptorTLV {
+
+  /** The minimum valid value in the oracle can sign */
+  def min: Vector[String]
```

Comment:
I think this can be removed now too

Reply:
So i think these are nice for the case where `overflow` and `underflow` have to be signed 

https://github.com/discreetlogcontracts/dlcspecs/blob/master/Oracle.md#digit-decomposition